Resources/Textures/_White/Clothing/Uniforms/Jumpsuit/prisonguard.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "license": "NC-BY-SA-4.0",
```

```suggestion
    "license": "CC-BY-NC-SA-4.0",
```
Я предполагаю что ты имел в виду эту лицензию